src/components/Logout.tsx

```diff
@@ -9,7 +9,7 @@ const Logout = (): JSX.Element => {
   const { t } = useTranslationHook()
 
   return (
-    <Button variant='danger' type='submit' onClick={onLogoutClick}>
+    <Button variant='danger' type='submit' className='badge' onClick={onLogoutClick}>
```

Comment:
why so small with a badge?
Why not a standard size?

Reply:
I resolved